app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, description, life):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.life = life
+
+    def to_dict(self):
+        return dict(
+            id=self.id,
+            name=self.name,
+            description=self.description,
+            life=self.life
+        )
+
+
+planets = [
+    Planet(1, "Earth", "Best planet", True),
+    Planet(2, "Saturn", "Got a ring on it", False),
+    Planet(3, "Mars", "We want to go there", False)
+]
+
+bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+def validate_planet(id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message":f"planet {id} invalid"}, 400))
+
+    for planet in planets:
+        if planet.id == id:
+            return planet
+
+    abort(make_response({"message":f"planet {id} not found"}, 404))
+
+
+@bp.route("", methods=["GET"])
+def get_planets():
```

Comment:
👍 

Consider using a list comprehension for generating the result list. This would be especially nice since you have the `to_dict` instance method.